src/DocumentFormat.OpenXml/Framework/OpenXmlNamespace.cs

```diff
@@ -245,6 +245,9 @@ private static string NormalizeNamespace(in OpenXmlNamespace ns)
             { "http://schemas.microsoft.com/office/spreadsheetml/2014/11/main", "x16", FileFormatVersions.Office2016 },
             { "http://schemas.microsoft.com/office/spreadsheetml/2015/02/main", "x16r2", FileFormatVersions.Office2016 },
             { "http://schemas.microsoft.com/office/word/2015/wordml/symex", "w16se", FileFormatVersions.Office2016 },
+            // Named 2013/main/command with "oac" ns prefix to avoid conflict with officedocument/2006/characteristics
+            { "http://schemas.microsoft.com/office/drawing/2013/main/command", "oac", FileFormatVersions.Office2016 },
```

Comment:
Where'd these come from?

Reply:
This is a new ns prefix and namespace added retro actively to 2016, 2019 and in 2021 to include moniker types that were not included a few years ago.